### PR TITLE
feat/AB#75545_See-if-possible-to-reduce-kendo-style-size

### PR DESCRIPTION
**Pull request by TaiKamilla**
# Description
Reducing the Theme Size by mapping the theme SCSS files to NPM packages

### Reduced the style in 22KB ( 17.18% )

Before:
```
Names     |  Raw Size | Estimated Transfer Size
main      |  11.53 MB |                 1.88 MB
styles    |   1.28 MB |               215.98 kB
scripts   | 923.23 kB |               249.29 kB
polyfills |  46.01 kB |                14.17 kB
runtime   |   6.07 kB |                 2.83 kB

Total     |  13.76 MB |                 2.35 MB
```
After:
```
Names     |  Raw Size | Estimated Transfer Size
main      |  11.53 MB |                 1.88 MB
styles    |   1.06 MB |               182.59 kB
scripts   | 923.23 kB |               249.29 kB
polyfills |  46.01 kB |                14.17 kB
runtime   |   6.07 kB |                 2.83 kB

Total     |  13.54 MB |                 2.32 MB
```

use this to build and compare the bundle size 
```
npx nx run back-office:build
```

## Useful links

- [Link to ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/75545)
- [Documentation](https://docs.telerik.com/themebuilder/partial-theme-build)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (refactor or addition to existing functionality)

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [ ] * I have commented my code, particularly in hard-to-understand areas
- [ ] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [ ] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by TaiKamilla** [comment]
I left the other styles there commented so it can be used if a new kendo component is added

**Comment by unai-reliefapp** [comment]
ey @TaiKamilla if you leave the styles just in the kendo theme file and delete them from the shared file they still work?
Could you give a try please?
Thanks
KR, Unai

**Comment by TaiKamilla** [comment]
> ey @TaiKamilla if you leave the styles just in the kendo theme file and delete them from the shared file they still work? Could you give a try please? Thanks KR, Unai

Hey @unai-reliefapp 
Yes, it works
But I thought that the "shared" or "style" folder was going to be removed
Was not of the libraries going to be removed?

**Comment by unai-reliefapp** [comment]
> > ey @TaiKamilla if you leave the styles just in the kendo theme file and delete them from the shared file they still work? Could you give a try please? Thanks KR, Unai
> 
> Hey @unai-reliefapp Yes, it works But I thought that the "shared" or "style" folder was going to be removed Was not of the libraries going to be removed?

First time hearing that
@AntoineRelief ?

But proper thing should be to keep all kendo related styles in the kendo folder of the styles lib I guess

**Comment by TaiKamilla** [comment]
Then probably I got it wrong
I heard something about a transition with the shared, and maybe I got it wrong
Probably is better to delete it 😅

**Comment by AntoineRelief** [comment]
> Then probably I got it wrong I heard something about a transition with the shared, and maybe I got it wrong Probably is better to delete it 😅

@TaiKamilla 
I think you misunderstood
we renamed one of the library ( safe ) to be shared, and this is already done
We are not planning on removing any library there

cc @unai-reliefapp

**Comment by unai-reliefapp** [comment]
Therefor please @TaiKamilla keep the kendo ones inside the kendo folder theme file 👍 
Thanks!

File: apps/back-office/project.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
         "styles": [
           "apps/back-office/src/styles.scss",
           "libs/shared/src/lib/style/styles.scss",
+          "libs/styles/src/lib/kendo/theme.scss",
```
Comment (unai-reliefapp) [inline apps/back-office/project.json]: ey @TaiKamilla  can you update the project.json of front-office and web-widget as they use kendo as well please?
Thanks!

**Review by unai-reliefapp — CHANGES_REQUESTED** [review]

**Comment by AntoineRelief** [comment]
@TaiKamilla 
I only changed the order in the styles arrays in project json files
Indeed, putting kendo before tailwind could create some conflicts

**Pull request merged**